test/jdk/javax/swing/plaf/synth/SynthButtonUI/6276188/bug6276188.java

```diff
@@ -79,6 +89,7 @@ public void run() {
             robot.waitForIdle();
             robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
             robot.waitForIdle();
+            robot.delay(2000);
```

Comment:
I wonder if syncing with `mousePressed` event handler on the button and adding a little delay after `mousePressed` is triggered would allow for smaller delay. Anyway, using an event handler will *guarantee*, the same handler is called in `ButtonUI`. We will still need a delay to allow the button UI delegate to handle the event and to repaint the button, yet that delay less than 2 seconds would be enough.

Reply:
Since the color change to Red will happen on MOUSE_PRESSED and MOUSE_OVER state, I wanted to give it more time to be in red color state so that robot can pick the color..
The failure was that it was picking up Green color even though the CI failure artifact shows background is Red

Reply:
Used CountDownLatch